dist.ini

```diff
@@ -0,0 +1,68 @@
+name    = Pod-Usage
+author  = Marek Rouchal <marekr@cpan.org>
+license = Perl_5
+copyright_holder = Marek Rouchal <marekr@cpan.org>
+copyright_year   = 2013
+
+[ReadmeAnyFromPod / Markdown_Readme]
+type = gfm
+source_filename = lib/Pod/Usage.pm
+filename = README.md
+location = root
+
+[Regenerate::AfterReleasers]
+plugin = Markdown_Readme
+
+[@Starter::Git]
+revision = 4
+managed_versions = 1
+installer = MakeMaker::Awesome
+-remove = MakeMaker::Awesome
+RewriteVersion.global = 1
+NextRelease.format = %-9v %{yyyy-MM-dd}d
+regenerate = Makefile.PL
+regenerate = META.json
+regenerate = README.md
+regenerate = LICENSE
+regenerate = t/00-report-prereqs.t
+Git::Check.allow_dirty = META.json
```

Comment:
The regenerate option for `[@Starter::Git]` already sets this

Reply:
fixed by 5d7bc49f